modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm

```diff
@@ -40,6 +40,10 @@
 	desc = "A fancy pair of high heels. Won't compensate for your below average height that much."
 	icon_state = "heels"
 
+/obj/item/clothing/shoes/high_heels/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('modular_skyrat/modules/modular_items/lewd_items/sounds/highheel3.ogg' = 1,'modular_skyrat/modules/modular_items/lewd_items/sounds/highheel2.ogg' = 1), 70)
```

Comment:
You don't need to literally just copy a sound file to have it play at a lower volume, I'm sure there's a better way to do it without having to do something like that, considering there's a volume setting on sounds anyway.

Reply:
> You don't need to literally just copy a sound file to have it play at a lower volume, I'm sure there's a better way to do it without having to do something like that, considering there's a volume setting on sounds anyway.

Oh, there is? I wasn't aware of that. Though given the file size being a few KB is that an issue?

Reply:
> > You don't need to literally just copy a sound file to have it play at a lower volume, I'm sure there's a better way to do it without having to do something like that, considering there's a volume setting on sounds anyway.
> 
> Oh, there is? I wasn't aware of that. Though given the file size being a few KB is that an issue?

Yes, it still is an issue, we don't have an infinite amount of resources for storing all these sounds and such, so if we're storing duplicates that are simply just using a different volume, we're just going to take the hit for performances.

Reply:
That's a good point. I'll begin that one soon!